--- elf_x86_64.x	2017-06-29 23:11:22.651936400 +0900
+++ space_x86_64.scr	2017-07-01 17:59:33.057048700 +0900
@@ -45,14 +45,15 @@
       PROVIDE_HIDDEN (__rela_iplt_start = .);
       *(.rela.iplt)
       PROVIDE_HIDDEN (__rela_iplt_end = .);
     }
   .init           :
   {
     KEEP (*(SORT_NONE(.init)))
+    . += 0x400000; /* テキスト領域の先頭に空きを作る */
   }
   .plt            : { *(.plt) *(.iplt) }
 .plt.got        : { *(.plt.got) }
 .plt.bnd        : { *(.plt.bnd) }
   .text           :
   {
     *(.text.unlikely .text.*_unlikely .text.unlikely.*)
